app/models/competition.rb

```diff
@@ -2088,13 +2069,13 @@ def set_wcif_events!(wcif_events, current_user)
 
   # Takes an array of partial Person WCIF and updates the fields that are not immutable.
   def update_persons_wcif!(wcif_persons, current_user)
-    registrations_relation = self.uses_microservice_registrations? ? self.microservice_registrations : self.registrations
+    registrations_relation = self.registrations
     registration_includes = [
       { assignments: [:schedule_activity] },
       :user,
       :wcif_extensions,
     ]
-    registration_includes << :registration_competition_events unless self.uses_microservice_registrations?
+    registration_includes << :registration_competition_events
```

Comment:
See my two remarks about inlining `registration_relation` and `registration_includes`.